ts/src/profiler.ts

```diff
@@ -331,8 +331,9 @@ export class Profiler extends ServiceObject {
             await SourceMapper.create(this.config.sourceMapSearchPath);
       } catch (err) {
         this.logger.error(
```

Comment:
Should this be a warning now?

Reply:
I think error is the right level because a feature has been disabled automatically. It is partially catastrophic for the user. But both are fine, slight preference for error.